crates/aranya-client-capi/src/imp/config.rs

```diff
@@ -239,15 +240,38 @@ impl Default for SyncPeerConfigBuilder {
 
 /// Configuration info when creating or adding a team in Aranya
 #[derive(Clone, Debug)]
-pub struct TeamConfig {}
+pub struct TeamConfig {
+    psk_idenitity: Option<Box<[u8]>>,
+    psk_secret: Option<Secret>,
+}
```

Comment:
The way this is currently defined would allow either a psk_identity or psk_secret to be defined without the other being defined. Either both should be defined or neither one should be defined.

Something like this would work better:
```suggestion
pub struct TeamConfig {
    psk: Option<Psk>,
}

struct Psk {
    psk_idenitity: Box<[u8]>,
    psk_secret: Secret,
}
```

As @elagergren-spideroak mentioned previously, we'll want to define a PSK for each hash algorithm which means we'll probably want to configure a `Vec` of PSKs here.